common/config/rush/command-line.json

```diff
@@ -18,16 +18,20 @@
       "shellCommand": "node eng/scripts/watch.js"
     },
     {
-      "commandKind": "global",
+      "commandKind": "bulk",
       "name": "test",
       "summary": "(CUSTOM) Runs all tests.",
-      "shellCommand": "node eng/scripts/npm-run.js test"
```

Comment:
Was there some logic we had that made sure you had test-offical if you had test? is this still used?

Reply:
oh, hmm, yeah, there was. I guess the risk is you add test but not test-official, but you can not add either, lol, so maybe not necessary to enforce?

Reply:
still used elsewhere but probably all could be bulk. One annoying thing I don't like is having to say --verbose to see output of bulk rush command.